inference-engine/ie_bridges/python/sample/object_detection_sample_ssd/README.md

```diff
@@ -61,9 +61,9 @@ To run the sample, you can use RMNet_SSD or other object-detection models. You c
 > 
 > The sample accepts models in ONNX format (.onnx) that do not require preprocessing.
 
-You can do inference of an image using a trained RMNet_SSD network on FPGA with fallback to CPU using the following command:
+You can do inference of an image using the OpenVINO&trade; toolkit person detection SSD model on FPGA with fallback to CPU using the following command:
```

Comment:
1) `using the OpenVINO&trade; toolkit person detection SSD model` --> `using the [person detection SSD model](@ref omz_models_intel_person_detection_retail_0013_description_person_detection_retail_0013) from the Open Model Zoo`

2) Since the FPGA plugin is no longer available, please replace FPGA with another plugin